m4/boost.m4

```diff
@@ -408,7 +408,7 @@ dnl until we succeed to link conftest.o in an executable.  The problem is
 dnl that the various TRY_LINK / COMPILE_IFELSE macros of Autoconf always
 dnl remove all the temporary files including conftest.o.  So the trick here
 dnl is to temporarily change the value of ac_objext so that conftest.o is
-dnl preserved accross tests.  This is obviously fragile and I will burn in
+dnl preserved across tests.  This is obviously fragile and I will burn in
```

Comment:
We pull this file from another repo (github.com/tsuna/boost.m4) every now and again, so this change will be reverted the next time we do.